packages/secrets/OVERVIEW.md

```diff
@@ -0,0 +1,45 @@
+# Secrets SDK
+
+Secrets for Zowe SDK is a new development package used in the following Zowe releases:
+- Zowe CLI, v7.18.0 and above
+- Zowe Explorer and Zowe Explorer API
+    - v2.10.0 and above (v2 LTS)
+    - v1.22.5 and above (v1)
+
+The `keyring` module in this package replaces node-keytar. Node-keytar was developed by Atom, and is no longer maintained as of December 15th, 2022.
+
+## Impact on End Users
+
+### Zowe CLI
+
+CLI users should not be affected by this change. It is intended to be a drop-in replacement for node-keytar, so existing credentials will still be accessible.
+
+### Zowe Explorer
+
+- **Most** Zowe Explorer users should not be affected by this change.
+- Some Zowe Explorer users may be affected by this change; specifically, users connecting over remote environments with Zowe Explorer might have to re-enter credentials. 
+    - Since keytar has been removed from VS Code for the next release, Zowe Explorer cannot access the local credential vault while the extension is running through a remote server. Those users will only have to re-enter their credentials once. 
```

Comment:
Suggesting a minor edit here, which is to define "those users": Users who run Zowe Explorer through a remote server need to re-enter their credentials only once. 
(Idk if my facts are correct, but hopefully you can see where I'm going w/ this.)